zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/autoconfigure/ObjectManagementAutoConfiguration.kt

```diff
@@ -39,6 +40,18 @@ import org.springframework.data.jpa.repository.config.EnableJpaRepositories
 @EntityScan("com.ritense.objectmanagement.domain")
 class ObjectManagementAutoConfiguration {
 
+    @Bean
+    @ConditionalOnMissingBean(ObjectManagementFacade::class)
+    fun objectManagementFacade(
```

Comment:
I'm missing a bit of context. Will this service be called inside a Camunda service task JUEL expression? If so, the `@ProcessBean` annotation should be added here

Reply:
Calling this in Camunda using JUEL is not in scope right now. The original intent for this class is to be usable from within the code without needing to call all this class's dependencies.